projects/ng-rule-editor/src/lib/rule-editor.service.ts

```diff
@@ -396,11 +396,24 @@ export class RuleEditorService {
     return fhir;
   }
 
+
+  /**
+   * Takes FHIR questionnaire definition and a linkId and returns a new FHIR
```

Comment:
Does it return new questionnaire or the same questionnaire with modified content?